sync/index.js

```diff
@@ -38,11 +38,9 @@ export function sendSynced(added) {
 }
 
 export async function syncMessage(added, ...data) {
-  let promises = []
-
   for (let i = 0; i < data.length - 1; i += 2) {
-    let action = data[i]
-    let meta = data[i + 1]
+    const action = data[i]
```

Comment:
Can you revert this change? It breaks CI since we use another code standard.

Reply:
yup, noted